src/navigator/Navigator.tsx

```diff
@@ -696,6 +704,17 @@ function ModalStackScreen() {
 }
 
 function RootStackScreen() {
+  let isSanctionedCountry = false
+  const userCountry = useSelector(userLocationDataSelector)
+  if (
+    userCountry.countryCodeAlpha2 === 'CU' ||
+    userCountry.countryCodeAlpha2 === 'IR' ||
+    userCountry.countryCodeAlpha2 === 'KP' ||
+    userCountry.countryCodeAlpha2 === 'SY'
```

Comment:
to make this easier to edit and maintain, i'd suggest to to add this as a variable in config.ts, like `const SANCTIONED_COUNTRIES = ['CU',....]` and create a selector that uses this config so that you can use it directly like `const isUserInSanctionedCountry = useSelector(userInSanctionedCountrySelector)` - what do you think?

the benefit of having these countries in an array, is you can do a simple check like `countries.includes(userCountry.countryCodeAlpha2)` in the selector

Reply:
Thanks for the suggestion! I did what you said :)